common/djangoapps/util/milestones_helpers.py

```diff
@@ -129,15 +143,18 @@ def get_prerequisite_courses_display(course_descriptor):
             required_course_descriptor = modulestore().get_course(course_key)
             prc = {
                 'key': course_key,
-                'display': get_course_display_name(required_course_descriptor)
+                'display': get_course_display_string(required_course_descriptor)
             }
             pre_requisite_courses.append(prc)
     return pre_requisite_courses
 
 
-def get_course_display_name(descriptor):
+def get_course_display_string(descriptor):
```

Comment:
Note to reviewers: 
Rationale for name change:
"display_name" is already a field on CourseDescriptor/CourseOverview that is different from what this function returns